enginetest/queries/load_queries.go

```diff
@@ -275,11 +274,11 @@ var LoadDataErrorScripts = []ScriptTest{
 	{
 		Name: "Load data with unknown columns throws an error",
 		SetUpScript: []string{
-			"create table loadtable(pk int primary key)",
+			"create table loadtable(pk int primary key, i int)",
 		},
 		Assertions: []ScriptTestAssertion{
 			{
-				Query:       "LOAD DATA INFILE './testdata/test1.txt' INTO TABLE loadtable FIELDS ENCLOSED BY '\"' (bad)",
+				Query:       "LOAD DATA INFILE './testdata/test1.txt' INTO TABLE loadtable FIELDS ENCLOSED BY '\"' (pk, fake_col, i)",
```

Comment:
The customer reported bug wasn't getting triggered in our existing tests for column name validation – to trigger this one, the bad column name must have an index that is less than the number of columns in the destination table's schema. Since these tests had only one column in the test table, any invalid column names after the first column name wouldn't trigger the same bug the customer hit.